recipe/meta.yaml

```diff
@@ -19,13 +19,16 @@ requirements:
   build:
     - python
     - pip
-    - numpy 1.10.4
+    - numpy 1.10.4  # [not win]
+    - numpy 1.11.*  # [win]
```

Comment:
we really do not support windows for MDAnalysis.Even if the package can be build it won't work correctly.

Reply:
That is not for me to decided, I don't really use `mdanalysis` :grimacing:

Reply:
True. But I wouldn't want to give users a broken package. I'm actually not sure why appveyor even tried to build this.

Reply:
B/c I removed the skip statement just for testing. Let me fix that for you.